index.html

```diff
@@ -0,0 +1,191 @@
+                        </div>
+                        <h3 class="header">LET'S CONNECT</h3>
+                    <div class="icons-container">
+                        <ul>
+                        <li><a href=""><img class="social-icons" src="images/twitter.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/medium.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/github.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angelist.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angellist.png" alt=""></a></li>
+                    </ul>
+                    </div>
+                <div class="resume">
+                    <img src="images/resume.png" alt="">
+                </div>
+                              
+                <div class="section5-head">
+                    <h1>Languages</h1>
+                </div>
+                <div class="lang">
+                    <img class="lango" src="images/down-arrow.png" alt="">
+                </div>  
+                <div class="languages">
+                    <div class="language-icon1">
+                        <img src="images/js.png" alt="">
+                        <a href=""><h4 class="icon1-head">javascript</h4></a>
+                    </div>
+                    <div class="language-icon2">
+                        <img src="images/html-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">HTML</h4></a>
+                    </div>
+                    <div class="language-icon3">
+                        <img src="images/css-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">CSS</h4></a>
+                    </div>
+                </div>   
+                <div class="section5-head2">
+                    <h1>Frameworks</h1>
+                    <img class="arrow-next1" src= "images/next.png" alt="">
+                </div>
+                <div class="section5-head3">
+                    <h1>Skills</h1>
+                    <img class="arrow-next2" src="images/next.png" alt="">
+                </div>
+
+<section class="first-section">
+    <h2>Contact Me</h2>
+    <p class="form-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mjvzkobj" method="post">
+        <input type="text" required placeholder="Alexendra loremipsum" class="name" name="Full name" maxlength="30">
+        <input type="email" placeholder="alexendra@gmail.com" class="email" name="Email" required>
+        <textarea id="text" cols="30" rows="10" maxlength="500" class="description" name="Your message" placeholder="Write your message here" required></textarea>
```

Comment:
Kindly, it is important not to confuse the PLACEHOLDER and NAME attributes, the first is only to tell the user what information to enter in the field, on the other hand the "NAME" attribute is the name of the field (it is like saying the name of the variable) , additionally the "ID" attribute is used in Js to carry out different events. I recommend you to use name for example name="first-name" or name="last-name", please make the corresponding adjustment.